Sources/Player/PlaybackEngine/Internal/Cache/DBCacheEntryDTO.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import GRDB
+
+// MARK: - CacheEntryType + DatabaseValueConvertible
+
+extension CacheEntryType: DatabaseValueConvertible {}
+
+// MARK: - DBCacheEntryDTO
+
+struct DBCacheEntryDTO: Codable, FetchableRecord, PersistableRecord {
```

Comment:
I was expecting the PR to kickstart a naming discussion :)

About the DB prefix, for me mainly I want to make it clear that it is the class for the GRDB implementation. there could be other like `UDCacheEntryDTO` for the UserDefaults one (which is how the current cache layer works)

I have also seen Entity used more for this, I had completely forgotten. Let's go with `DBCacheEntryEntity`?

Reply:
well, developers and naming... 💜 sometimes 💔. 😅
I'll share my opinion and the train of thoughts to justify it. 🤓 Might be related to each developer experience in the end and comes down to personal preference. 😅

I feel like the "DB" would make more sense as a prefix of the "entity" keyword instead of the name of the struct, since it's there as CacheEntry entity in GRDB or in User Defaults - something like: "CacheEntryDBEntity" and "CacheEntryUDEntity". For me, they are all related to CacheEntry, and then the entity type is different, one located in GRDB and another in user defaults.
And maybe not popular opinion but I would even have "GRDB" and "UserDefaults" instead of only "DB" and "UD" since just those two letters don't immediately convey "GRDB" and "User Defaults" _to me_, so IMO a longer name in this case makes the reader immediately understand what it is instead of have questions about it before looking how it is used and concluding what it is.